celt/arm/celt_neon_intr.c

```diff
@@ -86,8 +86,90 @@ void celt_float2int16_neon(const float * OPUS_RESTRICT in, short * OPUS_RESTRICT
       out[i] = FLOAT2INT16(in[i]);
    }
 }
+
+int opus_limit2_checkwithin1_neon(float *samples, int cnt)
+{
+   const float hardclipMin = -2.0f;
+   const float hardclipMax = 2.0f;
+
+   int i = 0;
+   int exceeding1 = 0;
+   int start = 0;
+
+#if defined(__ARM_NEON)
+   const int BLOCK_SIZE = 16;
+   const int FRAME_SIZE = (60000 / sizeof(float) / BLOCK_SIZE) * BLOCK_SIZE;
```

Comment:
I'm not sure I get this FRAME_SIZE. Looks like a cache optimization thing? In any case, given that Opus packets cannot be longer than 120 ms, I don't think that FRAME_SIZE value can ever be reached unless opus_pcm_soft_clip() gets called explicitly. Or did I miss something here?

Reply:
Originally, yes, it aimed to keep sample processing within cache limits, but we missed the maximum of the opus packet size.
I removed `FRAME_SIZE` and relevant logic handling it.